src/main/java/duke/exception/NotRecognizeException.java

```diff
@@ -0,0 +1,12 @@
+package duke.exception;
+
+/**
+ * Represents the specific exception that is related to the non-recognition.
+ *
+ * @author QIN GUORUI
+ */
+public class NotRecognizeException extends DukeException {
```

Comment:
Perhaps it is better to name this class as ```InvalidCommandException``` for better readability.
```suggestion
public class InvalidCommandException extends DukeException {
```